src/fides/api/ops/task/graph_task.py

```diff
@@ -587,16 +588,11 @@ def consent_request(self, identity: Dict[str, Any]) -> bool:
             )
             return False
 
-        consent_preferences: List[Consent] = [
-            Consent(**pref) for pref in self.resources.request.consent_preferences or []
-        ]
-
         output: bool = self.connector.run_consent_request(
             self.traversal_node,
             self.resources.policy,
             self.resources.request,
             identity,
-            consent_preferences,
```

Comment:
These are the old style consent preferences. I'm just pulling them off of the privacy request inside saasconnector.run_consent_request instead of passing them in here as a first step to getting rid of this old workflow in the future.